pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -424,6 +425,8 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 	configHash := h.Sum(nil)
 	same := bytes.Equal(configHash, appinstancePrevConfigHash)
 	if same {
+		// we need to apply filter in case of no changes in apps itself
+		filterAndPublishAppInstancesWithCurrentProfile(getconfigCtx)
```

Comment:
This need goes away if zedmanager applies the profile

Reply:
Done